airflow/config_templates/default_airflow.cfg

```diff
@@ -102,6 +102,18 @@ sql_alchemy_pool_size = 5
 # a lower config value will allow the system to recover faster.
 sql_alchemy_pool_recycle = 1800
 
+# How many seconds to wait before initial connect times out
+sql_alchemy_connect_timeout_seconds = 30
+
+# How many seconds to wait before statements time out
+sql_alchemy_statement_timeout_seconds = 30
```

Comment:
Both of these timeouts should default to 0 which would disable. Enabling this by default could be very surprising and potentially break things